transfer/transfer-06-http-data-flow/backend-service/src/main/java/org/eclipse/edc/handler/ReceiverHandler.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.handler;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import org.eclipse.edc.parser.HttpQueryParserBase;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.HashMap;
+
+/**
+ * A handler which is invoked to process HTTP exchanges.
+ * Each HTTP exchange is handled by one of these handlers.
+ */
+public class ReceiverHandler implements HttpHandler {
+
+    private final HttpQueryParserBase queryParser = new HttpQueryParserBase();
+
+    /**
```

Comment:
please avoid copy-and-paste comments, just describe what happens in the method. Same for the class comment, that could be avoided at all.